app/views.py

```diff
@@ -170,17 +153,19 @@ def view_all_lists():
 		if not isinstance(user_id, str):
 			results = []
 			# query parameters
+			#print ("hello")
 			q = request.args.get('q', None)  # this parameter contains the name of the list
-			limit = request.args.get('limit', 50)  # limits the number of records to 50 per page (optional)
-			page = request.args.get('page', 1)  # page one is default, but page can be passed as an argument (optional)
+			limit = request.args.get('limit', 50, type=int)  # limits the number of records to 50 per page (optional)
+			page = request.args.get('page', 1, type=int)  # page one is default, but page can be passed as an argument (optional)
 			if q is not None:
 				 #lists = Shopping_list.query.filter(
 				 #	Shopping_list.list.like("%" + q.strip() + "%")).\
 				 #	filter_by(id=id).paginate(page, limit, False).items
 
 				lists = Shopping_list.query.filter(Shopping_list.list.like("%"+q.strip()+"%")).filter_by(user_id=user_id).paginate(page, limit, False).items
 			else:
-				lists = Shopping_list.query.filter_by(id=id).paginate(page, limit, False).items
+				#print("here")

```

Comment:
ensure you remove all your debugging code "#print("here")"

Reply:
ok removed